runtime/Python3/src/antlr4/atn/ATN.py

```diff
@@ -58,7 +63,7 @@ def nextTokensNoContext(self, s:ATNState):
         if s.nextTokenWithinRule is not None:
             return s.nextTokenWithinRule
         s.nextTokenWithinRule = self.nextTokensInContext(s, None)
-        s.nextTokenWithinRule.readonly = True
+        s.nextTokenWithinRule.readOnly = True
```

Comment:
This is fixing what appears to be a capitalization typo. The `IntervalSet` class initializes this using camelCase instead of lowercase. Not sure which is preferred since I see both usages in the runtime.